Update_user.css

```diff
@@ -72,44 +111,108 @@ nav a{
 	height:680px;
 	width:100%;
 }
-
+/* Set up the footer */
 .footer{
-	background-color: #0C59CF;
-	grid-column-start: 1;
+	background-color: blue;
 	grid-column-end: span 4;
+	padding-top:25px;
 	text-align: center;
-	height:35px;
 	color:white;
-	padding-top:20px;
+	height:50px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
 }
 
+
+/* Re position of text in the div */
 h3{
 	margin-left:130px;
 	margin-top:30px;
 	
 }
 
+/* Re position of text in the div and change font-size, color, weight */
 p{
 	margin-left:130px;
 	margin-top:250px;
 	font-size:24px;
 	color:#0C59CF;
-	font-weight: bold;
+	font-family: 'Source Sans Pro', sans-serif;
+	font-weight:600;
 }
 
 #us{
 	height:30px;
 	width: 240px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
 }
 
 #pa{
 	height:30px;
 	width: 240px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
 }
 
+/* Set up the login buttom*/
 #login{
 	height:25px;
 	width: 80px;
 	color:white;
 	background-color:#0C59CF;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
+}
+
+/* Set up the text display correct*/
+.Connected{
```

Comment:
Create an class called Connected and when it is successfully update the new password then it will display and this class is use to setting who that text will look like. Eg(font, color, size)